install.sh

```diff
@@ -325,6 +325,10 @@ if [[ ! -f "$RELAY_CREDENTIALS_JSON" ]]; then
   echo "Relay credentials written to $RELAY_CREDENTIALS_JSON"
 fi
 
+
+./install/geoip.sh
```

Comment:
I know install.sh is getting a bit unwieldy but does splitting geoip part only help much?

Reply:
Yes. By splitting out the geoip part I am able to [test it in isolation](https://github.com/getsentry/onpremise/pull/766#issuecomment-743247145).

Reply:
I don't either. :) I think we should refactor install.sh to make it more modular and therefore more testable ... and we should test it! 😁 I don't see this bloating the root director with many `.sh` files, you can see here I split the subscript into an `install/` directory. Yes we can and will share components between these scripts ... again, one step at a time.